service/collector_test.go

```diff
@@ -63,6 +63,56 @@ service:
       exporters: [otlp]
 `
 
+// colTelemetryTest initializes telemetry without affecting the global sync.Once.
+type colTelemetryTest struct {
```

Comment:
Why is this needed instead of using directly your own instance of colTelemetry?

Reply:
There was no need, fixed :)